amundsen_application/static/js/components/TableDetail/RequestMetadataForm/index.tsx

```diff
@@ -0,0 +1,110 @@
+import * as React from 'react';
+import './styles.scss';
+
+import { sendNotification } from 'ducks/notification/api/v0';
+import { NotificationType } from 'interfaces/Notifications';
+import { AvatarLabelProps } from 'components/common/AvatarLabel';
+import { GlobalState } from 'ducks/rootReducer';
+import { connect } from 'react-redux';
+
+interface StateFromProps {
+  userEmail: string;
+  tableName: string;
+  tableOwners: Array<String>;
+}
+
+type RequestMetadataProps = StateFromProps;
+
+interface RequestMetadataState {
+  isOpen: boolean,
+}
+
+export class RequestMetadataForm extends React.Component<RequestMetadataProps, RequestMetadataState> {
+  public static defaultProps: RequestMetadataProps = {
+    userEmail: '',
+    tableName: '',
+    tableOwners: [],
+  };
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isOpen: true,
+    };
+  }
+
+  toggle = () => {
+    this.setState({ isOpen: !this.state.isOpen });
+  }
+
+  submitForm = (event) => {
+    event.preventDefault();
+    const form = document.getElementById("RequestForm") as HTMLFormElement;
+    const formData = new FormData(form);
+    const recipients = String(formData.get('recipients')).split(",");
```

Comment:
Pretty sure I need to cast these as strings because formData.get() returns a FormDataEntryValue object, not a string

Reply:
A `FormDataEntryValue` is documented as a "A string which represents a single value from a set of FormData key-value pairs" https://developer.mozilla.org/en-US/docs/Web/API/FormDataEntryValue...I wonder what's going on.

Do you get a type error when using `formData.get('recipients').split(",")` or an actual JS error?

Reply:
Im getting a type error, without the casting I get the error: Property 'split' does not exist on type 'FormDataEntryValue'.

Reply:
Did a bit more searching. That doc is misleading -__-, I looked at some code and it seems TypeScript defines `FormDataEntryValue` as being either `string | File`. So because it can be either, that would be why the code won't let you treat it as a string. 

However...we know it's going to be a string because we control the logic that generated the value downcast them as `string`.
```
...
const recipients = formData.get('recipients') as string;
const recipientsList = recipients.split(",");
const sender = formData.get('sender') as string;
....
```